custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -17,9 +17,9 @@
 TEST_CASE_TYPE = 'test'
 
 
-def validate_number(string_value):
+def validate_phone_number(string_value):
     if string_value is None or string_value.strip() == '':
-        return None
+        return ''
     else:
         return int(string_value)
```

Comment:
(also might want to update the phonenumbers ref if you end up using it to format the numbers, looks like it hasn't been updated in a year or so)